### PR TITLE
Updated material to run singularity w/o root privileges

### PR DESCRIPTION
**Pull request by mambelli**
Updated material to run singularity w/o root privileges
- Updated install/setup instructions
- Using fakeroot it is possible to avoid invoking as root
Fixed also some remaining unclear references to Sylabs and clarified references to the library

2022-11-1 Hackathon contribution
This solves issue #47

### CODE REVIEW

File: _episodes/02-running-containers.md [hunk 1]
```diff
@@ -27,9 +27,43 @@ singularity --help
...
 
-The [Singularity Container Library](https://cloud.sylabs.io/library) is the official image registry.
-Images built by users are accessible using the CLI, and images become containers at runtime.
+Apptainer/Singularity can store, search and retrieve images in registries.
+Images built by other users can be accessible using the CLI, can be pulled down, and become containers at runtime.
 
+Sylabs, the developers of one Singularity flavor, hosts a public image registry, the
+[Singularity Container Library](https://cloud.sylabs.io/library) where many user built images are available.
+
+The Linux Foundation singularity, apptainer does not point by default to the Sylab registry as previous versions did.
```
Comment (michmx) [inline _episodes/02-running-containers.md]: Thanks for the additions! I feel this first sentence is not clear in the sense of calling Apptainer "The Linux Foundation singularity".  What about using "The Linux Foundation flavor, Apptainer, does not point..."

**Comment by michmx** [comment]
Thanks a lot, @mambelli ! I didn't know about `--fakeroot`, amazing!

File: _episodes/05-definition-files.md [hunk 2]
```diff
@@ -142,26 +146,33 @@ Info in <TCanvas::Print>: png file rf101_basics.png has been created
...
+If you don't have a DISPLAY setup, Root may complain. Ignore the error messages, the image will be created anyway,
+
+Here we have covered the basics with a few examples focused to HEP software.
+Check the [Apptainer docs](https://apptainer.org/docs/user/main/build_a_container.html) to see all the available
+options and more details related to the container creation.
+
 
 > ## Deploying your containers
-> Keep in mind that, while build a container requires superuser permissions, the execution can be performed anywhere.
+> Keep in mind that, while building a container may be time consumin, the execution can be immediate and anywhere your image is available.
```
Comment (michmx) [inline _episodes/05-definition-files.md]: Typo: `consumin` -- > `consuming`

File: index.md [hunk 3]
```diff
@@ -5,18 +5,17 @@ permalink: index.html  # Is the only page that doesn't follow the pattern /:path
 ---
 Apptainer (formerly known as Singularity) is a free and open-source container platform that allows you to create and run applications in isolated images (also called "containers") in a simple, portable, fast, and secure manner. It performs operating system level virtualization known as containerization. Many container platforms are available, but Apptainer is designed to bring containers and reproducibility to the scientific community and High-Performance Computing (HPC) use cases. Using Apptainer, developers can work in reproducible environments of their choice and design, and these complete environments can be easily copied and executed on other platforms.
 
-An introduction to Singularity/Apptainer, its motivations and applications in HEP.
+This is An introduction to Singularity/Apptainer, its motivations and applications in HEP.
```
Comment (michmx) [inline index.md]: A minor thing, but here capital 'A' must be lowercase.

**Review by michmx — CHANGES_REQUESTED** [review]
Thanks a lot for this amazing improvement in the material. I just added a few minor comments and a typo that I found.

**Comment by mambelli** [comment]
I fixed all comments and rebased the PR

File: index.md (hunk 3 above)
Comment (mambelli) [reply index.md]: done

File: _episodes/05-definition-files.md (hunk 2 above)
Comment (mambelli) [reply _episodes/05-definition-files.md]: done

File: _episodes/02-running-containers.md (hunk 1 above)
Comment (mambelli) [reply _episodes/02-running-containers.md]: done

**Review by michmx — APPROVED** [review]

**Comment by klieret** [comment]
@all-contributors please add @mambelli for content

**Pull request merged**